cdap-ui/app/services/constants.js

```diff
@@ -48,8 +58,38 @@ angular.module(PKG.name + '.services')
         'httpcallback': 'Make HTTP Call'
       }
     },
+    'pluginTypeToID': {
+      'transform': 'Transform',
+      'batchsource': 'BatchSource',
+      'batchsink': 'BatchSink',
+      'batchaggregator': 'Aggregate',
+      'realtimesink': 'RealtimeSink',
+      'realtimesource': 'RealtimeSource',
+      'sparksink': 'Model',
+      'sparkcompute': 'Compute',
+      'batchjoiner': 'Join',
+      'action': 'Action',
+      'streamingsource': 'BatchSource',
+      'windower': 'Windower'
```

Comment:
I'm not sure I understand why we need to differentiate to pluginTypeToID? if we want uniqueness the pluginType is already unique right?